distributed/comm/__init__.py

```diff
@@ -20,7 +20,7 @@ def _register_transports():
     try:
         from . import ucx
     except ImportError:
-        pass
+        print("UCX Import Error!")
```

Comment:
Two issues with this:

1.  We probably can't use print statements in library code.  If we wanted to do something here we would probably use the `logging` or `warnings` modules
2.  UCX won't be relevant for most Dask users (99% of Dask use today has nothing to do with RAPIDS or GPUs) so we'll need to optimize for that case.  Users without UCX installed shouldn't be troubled by a message like this.